test/specs/Conformance-test.js

```diff
@@ -50,7 +50,7 @@ describe('Conformance', () => {
         if (_.has(_meta, 'parent')) {
           describe('parent', () => {
             it('matches some component name', () => {
-              expect(_.pluck(componentInfo, 'constructorName')).to.contain(_meta.parent)
+              expect(_.map(componentInfo, 'constructorName')).to.contain(_meta.parent)
```

Comment:
Heh, there were lots of other [breaking lodash changes](https://github.com/lodash/lodash/wiki/Changelog#v400) but I mostly tried to remove them in favor of ES201* features like spread and named destructuring.